framework/core/hpp_physical_device.cpp

```diff
@@ -138,5 +138,12 @@ vk::PhysicalDeviceFeatures &HPPPhysicalDevice::get_mutable_requested_features()
 	return requested_features;
 }
 
+vk::FormatProperties HPPPhysicalDevice::get_format_properties(vk::Format format) const
+{
+	vk::FormatProperties format_properties;
+	handle.getFormatProperties(format, &format_properties);
+	return format_properties;
```

Comment:
Just call
`	return handle.getFormatProperties(format);
`
Or maybe even skip this get_format_properties-function and just use
`gpu.get_handle().getFormatProperties(format)`, instead

Reply:
I'll remove it, and use your second suggestion